src/applications/static-pages/createCoronavirusChatbot.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default (store, widgetType) => {
+  // Derive the element to render our widget.
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  ReactDOM.render(
+    <Provider store={store}>
+      <iframe
+        id={'va-coronavirus-chatbot'}
+        title="VA Coronavirus chatbot"
+        src="https://va-covid19-chatbot-dev.azurewebsites.net/chatbot.html"
+        style={{ height: `${800}px`, width: `${100}%` }}
+      />
+    </Provider>,
+    root,
+  );
+
+  /*
+  const defaultLocale = 'en-US';
+  const localeRegExPattern = /^[a-z]{2}(-[A-Z]{2})?$/;
+  let chatBotScenario = 'unknown';
+
+  function extractLocale(localeParam) {
+    if (localeParam === 'autodetect') {
+      return navigator.language;
+    }
+
+    // Before assigning, ensure it's a valid locale string (xx or xx-XX)
+    if (localeParam.search(localeRegExPattern) === 0) {
+      return localeParam;
+    }
+    return defaultLocale;
+  }
+
+  function getUserLocation(callback) {
+    navigator.geolocation.getCurrentPosition(
+      position => {
+        const latitude = position.coords.latitude;
+        const longitude = position.coords.longitude;
+        const location = {
+          lat: latitude,
+          long: longitude,
+        };
+        callback(location);
+      },
+      error => {
+        // user declined to share location
+        // eslint-disable-next-line no-console
+        console.log(`location error:${error.message}`);
+        callback();
+      },
+    );
+  }
+
+  function initBotConversation() {
+    if (this.status >= 400) {
+      alert(this.statusText);
+      return;
+    }
+    // extract the data from the JWT
+    const jsonWebToken = this.response;
+    const tokenPayload = JSON.parse(atob(jsonWebToken.split('.')[1]));
+    const user = {
+      id: tokenPayload.userId,
+      name: tokenPayload.userName,
+      locale: tokenPayload.locale,
+    };
+    let domain = undefined;
+    if (tokenPayload.directLineURI) {
+      domain = `https://${tokenPayload.directLineURI}/v3/directline`;
+    }
+    const botConnection = window.WebChat.createDirectLine({
```

Comment:
I'm assuming this commented code is what's executed inside of the `iframe`. Ideally we'd be able to execute that directly, so I'm looking at the global dependencies to see how portable the code is. What is `window.WebChat`?

Reply:
Quick Google - it looks like it can be installed via NPM - https://www.npmjs.com/package/botframework-webchat

Reply:
I believe that's the webchat component that's offered by the Microsoft Bot Framework. This [link](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-webchat-overview?view=azure-bot-service-4.0#integrate-with-react) looks helpful for React-ifying the webchat component

Reply:
I'm looking into this as well: [another reference for React with this webchat component](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/01.getting-started/e.host-with-react/)

Reply:
I think you can install this via `yarn add https://www.npmjs.com/package/botframework-webchat`